test/functional/tests/writeback/test_writeback_signals.py

```diff
@@ -0,0 +1,251 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+from time import sleep
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from internal_plugins.scsi_debug import ScsiDebug
+from storage_devices.device import Device
+from storage_devices.disk import DiskType, DiskTypeSet
+from core.test_run import TestRun
+from test_tools.disk_utils import Filesystem
+from test_tools.fs_utils import create_random_test_file
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+
+mount_point = "/mnt/cas"
+log_msg = "/var/log/messages"
+
+
+@pytest.mark.parametrize("cache_mode", [CacheMode.WB, CacheMode.WO])
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_flush_signal_core(cache_mode):
+    """
+        title: Test for FLUSH signals to core device in mode with lazy writes.
```

Comment:
tests checking FLUSH often checked FUA signals as well - isn't it the case here? if so use plugin "scsi_debug_fua_signals"